_events/2019-09-09-openhouse.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Open House \#1
+featured:
+location: Our house
+address: >-
+  114 N. Orchard St.
+  Madison, WI 53715
+---
+
+Join us for our first Open House of the fall 2019 semester! Starting at 6 pizza and drinks will be available. Come any time and stay as long as you want. It’s guaranteed to be a packed house and a great chance to meet the members.
```

Comment:
I would remove 'It's guranteed to be a packaged house'. It's not always 'packed'